kafka-admin-client/src/test/java/io/mosip/kafkaadminclient/MosipKafkaAdminClientTest.java

```diff
@@ -0,0 +1,100 @@
+		Set<String> sampleStrings = new HashSet<>();
+		sampleStrings.add("test");
+		Mockito.when(kafkaFuture.get()).thenReturn(sampleStrings);
+		Mockito.when(listTopicsResult.names()).thenReturn(kafkaFuture);
+		Mockito.when(adminMock.listTopics(Mockito.any())).thenReturn(listTopicsResult);
+		MosipKafkaAdminClient mosipKafkaAdminClient = new MosipKafkaAdminClient("test");
+		Assert.assertTrue(mosipKafkaAdminClient.isTopicsPresent("test"));
+	} 
+    
+    @SuppressWarnings("unchecked")
+	@Test
+	public void createTopicsExceptionTest() throws InterruptedException, ExecutionException {
+    	PowerMockito.mockStatic(Admin.class);
+		KafkaAdminClient adminMock = mock(KafkaAdminClient.class);
+		PowerMockito.when(Admin.create(any(Properties.class))).thenReturn(adminMock);
+		CreateTopicsResult createTopicsResult = Mockito.mock(CreateTopicsResult.class);
+		Map<String, KafkaFuture<Void>> testMap = new HashMap<>();
+		KafkaFuture<Void> kafkaFuture = Mockito.mock(KafkaFuture.class);
+		testMap.put("test-topic", kafkaFuture);
+		Mockito.when(kafkaFuture.get()).thenThrow(ExecutionException.class);
+		Mockito.when(createTopicsResult.values()).thenReturn(testMap);
+		Mockito.when(adminMock.createTopics(Mockito.any())).thenReturn(createTopicsResult);
+		MosipKafkaAdminClient mosipKafkaAdminClient = new MosipKafkaAdminClient("test-topic");
+		Assert.assertThrows(ExecutionException.class, () -> mosipKafkaAdminClient.createTopic("test-topic"));
+    }
+    
+    @SuppressWarnings("unchecked")
+	@Test
+	public void createTopicsTest() throws Exception {
+    	PowerMockito.mockStatic(Admin.class);
+		KafkaAdminClient adminMock = mock(KafkaAdminClient.class);
+		PowerMockito.when(Admin.create(any(Properties.class))).thenReturn(adminMock);
+		CreateTopicsResult createTopicsResult = Mockito.mock(CreateTopicsResult.class);
+		Map<String, KafkaFuture<Void>> testMap = new HashMap<>();
+		KafkaFuture<Void> kafkaFuture = Mockito.mock(KafkaFuture.class);
+		testMap.put("test-topic", kafkaFuture);
+		Mockito.when(kafkaFuture.get()).thenReturn(null);
+		Assert.assertEquals(kafkaFuture.get(), null);
+		Mockito.when(createTopicsResult.values()).thenReturn(testMap);
+		Mockito.when(adminMock.createTopics(Mockito.any())).thenReturn(createTopicsResult);
+		MosipKafkaAdminClient mosipKafkaAdminClient = new MosipKafkaAdminClient("test-topic");
+		mosipKafkaAdminClient.createTopic("test-topic");
+    }
+    
+    @SuppressWarnings("unchecked")
+	@Test
+	public void describeTopicTest() throws Exception {
+    	PowerMockito.mockStatic(Admin.class);
+		KafkaAdminClient adminMock = mock(KafkaAdminClient.class);
+		PowerMockito.when(Admin.create(any(Properties.class))).thenReturn(adminMock);
+		DescribeTopicsResult result = Mockito.mock(DescribeTopicsResult.class);
+		KafkaFuture<Map<String, TopicDescription>> kafkaFutureVal = Mockito.mock(KafkaFuture.class);
+		Map<String, TopicDescription> resultMap = new HashMap<>();
```

Comment:
Can we fill some data here, to make sure assert is giving the same ?

Reply:
Done.